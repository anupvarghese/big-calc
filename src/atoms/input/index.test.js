// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import serializer from 'jest-glamor-react';

import Input from './';

expect.addSnapshotSerializer(serializer);

describe('input', () => {
  it('renders Input', () => {
    const button = renderer
      .create(
        <Input style={{ color: 'red' }} onChange={() => {}} value="Hello" />,
      )
      .toJSON();
    expect(button).toMatchSnapshot();
  });
});
