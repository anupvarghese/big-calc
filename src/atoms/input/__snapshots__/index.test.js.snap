// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input renders Input 1`] = `
<input
  onChange={[Function]}
  style={
    Object {
      "color": "red",
    }
  }
  value="Hello"
/>
`;
